db/models.py

```diff
@@ -19,34 +29,78 @@ def __str__(self) -> str:
 class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
-    actors = models.ManyToManyField(to=Actor, related_name="movies")
-    genres = models.ManyToManyField(to=Genre, related_name="movies")
+    genres = models.ManyToManyField(Genre)
+    actors = models.ManyToManyField(Actor)
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(null=True)
```

Comment:
Code Style: The `created_at` field in the `Order` model is nullable. It's better to make it non-nullable and set a default value to `timezone.now`. This way, when a new order is created, the current time will be automatically used if no other value is provided.